apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -88,7 +87,14 @@ private void addCommandData(StringBuilder dbStatement, CommandData commandData)
         if (RedissonPluginConfig.Plugin.Redisson.TRACE_REDIS_PARAMETERS && commandData.getParams() != null) {
             for (Object param : commandData.getParams()) {
                 dbStatement.append(DELIMITER_SPACE);
-                String paramStr = param instanceof ByteBuf ? QUESTION_MARK : String.valueOf(param.toString());
+                String paramStr;
+                if (param instanceof ByteBuf) {
```

Comment:
Could you provide what is a bytebuf?

Reply:
> Could you provide what is a bytebuf?

I get results like "EXISTS [B@1ce73d9a",But I'm not sure it's always like that

`if (param instanceof ByteBuf)` may always be `false`

Reply:
How does the latest look like? Could you post some examples?